stories/components/people-list/people-list.scss

```diff
@@ -7,29 +7,26 @@
 }
 
 .bix-filter__dropdown {
+  margin-block-start: units(5);
```

Comment:
> [!TIP]
> These `block-start`, `block-end`, `inline-start`, `inline-end` are logical properties that are recommended over the traditional `margin-top`, etc.
>
>It's supposed to help with internationalization.

Reply:
This is very interesting. I also just went down a rabbit hole about aria live polite and roles. Very cool stuff.